### ISSUE TITLE
`import* as React from 'react'` should throw a whitespace lint error

### ISSUE THREAD

**Issue by mohsen1** [palantir/tslint#1584]
### Bug Report
- **TSLint version**: `"tslint": "^3.15.1",`
- **TypeScript version**: `"typescript": "^2.0.3",`
- **Running TSLint via**: VSCode
#### TypeScript code being linted

``` ts
import* as React from 'react'
```

with `tslint.json` configuration:

``` json
{
    "extends": [
        "tslint:latest"
    ],
    "rules": {
      "quotemark": [
        true, "single", "avoid-escape"
      ]
    }
}

```
#### Actual behavior

No lint error
#### Expected behavior

Whitespace lint error
